nucypher/blockchain/eth/interfaces.py

```diff
@@ -465,16 +465,11 @@ def __log_transaction(self, transaction_dict: dict, contract_function: ContractF
 
     @validate_checksum_address
     def build_transaction(self,
```

Comment:
The name doesn't seem right. This is more a function for "payloads" (following the language we use in the codebase). What do you think if we change it to `build_payload` or something along those lines?

Reply:
Yes - you are correct is not really a complete transaction.  `build_payload` is fine. :+1:

Reply:
:heavy_check_mark: